server/src/main.rs

```diff
@@ -167,6 +168,36 @@ fn main() {
             format!("Restate {}", build_info::RESTATE_SERVER_VERSION)
         );
         let _ = writeln!(&mut stdout, "{:^40}", "https://restate.dev/");
+        if config.has_role(Role::Admin) {
+            let _ = writeln!(
+                &mut stdout,
+                "{:^40}",
+                format!(
+                    "Admin: {}",
+                    config
+                        .admin
+                        .advertised_admin_endpoint
+                        .as_ref()
+                        .expect("is set")
+                )
+            );
+        }
+
+        if config.has_role(Role::HttpIngress) {
+            let _ = writeln!(
+                &mut stdout,
+                "{:^40}",
+                format!(
+                    "Http Ingress: {}",
+                    config
+                        .admin
+                        .advertised_ingress_endpoint
```

Comment:
I am not sure why the `advertised_ingress_endpoint` is under `admin` and not `ingress` config! I assume because it's mainly use to tell UI which ingress url to use but imho it still makes more sense to move this to ingress.

Reply:
I'm torn on this. On one hand, it's weird that it's in admin, but on the other, it also doesn't impact ingress role at all.

Reply:
I agree with @pcholakov on this. If you all agree I can move it to ingress which will make more sense tbh